CHANGELOG.md

```diff
@@ -3,7 +3,21 @@
 
 Full documentation for MIOpen is available [here](https://rocm.docs.amd.com/projects/MIOpen/en/latest/)
 
-## MIOpen-3.2.0 for ROCm 6.2.0
+## MIOpen 3.3.0 for ROCm 6.3.0
+### Added
+- [RNN] LSTM fwd
```

Comment:
Should this be forward or forwarding? Or is fwd okay?

Reply:
I think fwd is ok